### PR TITLE
[3.0] Revert removal of framework.csrf_protection section

### PR DESCRIPTION
**Pull request by wouterj**
| Q | A |
| --- | --- |
| Bug fix? | [yes |
| New feature? | [yes |
| BC breaks? | [yes |
| Deprecations? | [yes |
| Tests pass? | yes |
| Fixed tickets | #16508 |
| License | MIT |
| Doc PR | - |

This section was incorrectly removed from Symfony 3, only the `field_name` setting was removed. Disabling/enabling CSRF seperately from the froms is not deprecated and should not be removed.

/cc @symfony/deciders please merge quickly, it's holding up bundles with functional tests wanting to support to Symfony 3

### CODE REVIEW

**Comment by fabpot** [comment]
@WouterJ Can you have a looks at the broken tests?

**Comment by WouterJ** [comment]
@fabpot build should be fixed now

File: src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd [hunk 1]
```diff
@@ -46,6 +47,11 @@
         <xsd:attribute name="field-name" type="xsd:string" />
     </xsd:complexType>
 
+    <xsd:complexType name="csrf_protection">
+        <xsd:attribute name="enabled" type="xsd:boolean" />
+        <xsd:attribute name="field-name" type="xsd:string" />
```
Comment (aitboudad) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd]: the `field_name` should be removed here

Comment (WouterJ) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd]: fixed

**Comment by fabpot** [comment]
Thank you @WouterJ.

**Pull request merged**